web/pdf_page_view.js

```diff
@@ -468,19 +472,15 @@ class PDFPageView {
     if (!this.textLayer) {
       return;
     }
-    this.structTreeLayer ||= new StructTreeLayerBuilder();
 
-    const tree = await (!this.structTreeLayer.renderingDone
-      ? this.pdfPage.getStructTree()
-      : null);
-    const treeDom = this.structTreeLayer?.render(tree);
-    if (treeDom) {
+    const treeDom = await this.structTreeLayer.render();
+    if (treeDom?.parentNode !== this.canvas) {
```

Comment:
Won't this attempt to insert a `null` structTree into the DOM, which I don't believe makes sense?
```suggestion
    if (treeDom && treeDom.parentNode !== this.canvas) {
```